themes/SuiteP/css/Dusk/style.css

```diff
@@ -12373,7 +12373,7 @@ div.qtip.qtip-default {
 
 .modal-content .search_form .view select,
 .modal-content .search_form .view select#saved_search_select {
-  border: 2px solid #A5E8D6; }
+  border: 2px solid #c4e3f5; }
```

Comment:
Using a hard-coded colour.

Reply:
it's not hard coded. I will minify these css files.